src/components/AstroPiModel/AstroPiModel.scss

```diff
@@ -246,15 +247,15 @@
 
       &-panel {
         &__control {
-          box-shadow: inset 0 0 var(--spacing-2) $rpf-grey-100;
+          box-shadow: inset 0 0 $space-3 $rpf-grey-100;
           background-color: $rpf-grey-50;
```

Comment:
As above - `space-3` replaces `spacing-2`: is that okay?

Reply:
As above!